### PR TITLE
feat: add Android Dark Mode Support plugin

### PR DESCRIPTION
**Pull request by robingenz**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] The changes have been tested successfully.
- [ ] A changeset has been created (`npm run changeset`).
- [ ] I have read and followed the [pull request guidelines](https://capawesome.io/contributing/pull-requests/).

---

See https://github.com/capawesome-team/capacitor-android-dark-mode-support/pull/13

### CODE REVIEW

**Comment by robingenz** [comment]
@piotr-cz Please give this pre release a try and let me know if everything works as expected:

```
npm i https://pkg.pr.new/capawesome-team/capacitor-plugins/@capawesome/capacitor-android-dark-mode-support@308
```

File: packages/android-dark-mode-support/rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export default {
+  input: 'dist/esm/index.js',
+  output: [
+    {
+      file: 'dist/plugin.js',
+      format: 'iife',
+      name: 'capacitorTorch',
```
Comment (piotr-cz) [inline packages/android-dark-mode-support/rollup.config.js]: iife name is wrong, I guess it should be `capacitorAndroidDarkModeSupport`

Comment (robingenz) [reply packages/android-dark-mode-support/rollup.config.js]: Good catch, thanks!

**Comment by robingenz** [comment]
@piotr-cz This is the new pre release:

```
npm i https://pkg.pr.new/capawesome-team/capacitor-plugins/@capawesome/capacitor-android-dark-mode-support@48e2d48
```

**Comment by piotr-cz** [comment]
This is pretty tricky to test, because one needs a device with WebView in a version that
- [supports](https://caniuse.com/prefers-color-scheme) `prefers-color-scheme media query`: >= 76,
- and does not [support native Dark mode](https://github.com/ionic-team/capacitor/issues/4677#issuecomment-1126274884): < 100

...and downgrading WebViews [is problematic](https://android.stackexchange.com/questions/139413/android-system-webview-installed-but-not-used-by-apps/233282#233282)

But I can confirm that I was able to replace `@robingenz/capacitor-android-dark-mode-support` with this version, build project and run on an emulator.

Given that there are no changes to code - yes, everything works as expected.

**Comment by robingenz** [comment]
@piotr-cz Thank you!

**Pull request merged**